internal/testutil/acc/cluster.go

```diff
@@ -0,0 +1,42 @@
+package acc
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+)
+
+func GetClusterInfo(projectID string) (clusterName, clusterNameStr, clusterTerraformStr string) {
+	// Allows faster test execution in local, don't use in CI
+	clusterName = os.Getenv("MONGODB_ATLAS_CLUSTER_NAME")
+	if clusterName != "" {
+		clusterNameStr = fmt.Sprintf("%q", clusterName)
+	} else {
+		clusterName = acctest.RandomWithPrefix("test-acc")
+		clusterNameStr = "mongodbatlas_cluster.test_cluster.name"
+		clusterTerraformStr = fmt.Sprintf(`
```

Comment:
consider using https://pkg.go.dev/embed for external files, it will let you leverage code highlight by not having it as a string here

Reply:
nice feature I was not aware of, thanks for sharing. For this case I am more hesitant because the string does have parameters, and seeing which parameters are being passed together with the content of the string is convenient.